tools/create-yorkie-app/index.ts

```diff
@@ -262,15 +261,15 @@ function copyDir(srcDir: string, destDir: string) {
 
 function isEmpty(path: string) {
   const files = fs.readdirSync(path);
-  return files.length === 0 || (files.length === 1 && files[0] === '.git');
+  return !files.length || !files.filter((file) => /^[^.]+/.test(file)).length;
 }
 
 function emptyDir(dir: string) {
   if (!fs.existsSync(dir)) {
     return;
   }
   for (const file of fs.readdirSync(dir)) {
-    if (file === '.git') {
+    if (/^\./.test(file)) {
```

Comment:
I think we can replace these regular expressions with `String.startsWith` method

Reply:
That makes the code much simpler. Thank you for your suggestion and a regex tip!